### PR TITLE
🧹 add k8s services to discovery

### PR DESCRIPTION
**Pull request by vjeffrey**
Screenshot 2024-06-10 at 16 16 52

### CODE REVIEW

File: providers/k8s/resources/discovery.go [hunk 1]
```diff
@@ -933,6 +996,10 @@ func createPlatformData(objectKind, runtime string) (*inventory.Platform, error)
 		platformData.Family = append(platformData.Family, "k8s-workload")
 		platformData.Name = "k8s-cronjob"
 		platformData.Title = "Kubernetes CronJob"
+	case "Service":
+		platformData.Family = append(platformData.Family, "k8s-workload")
```
Comment (czunker) [inline providers/k8s/resources/discovery.go]: A `service` is not really a workload. Perhaps we need a new family for this.

Comment (czunker) [reply providers/k8s/resources/discovery.go]: @imilchev Any ideas? Or should we go with `k8s-service` as its own family, like in `k8s-ingress`.

**Comment by czunker** [comment]
Gave this a try. LGTM:
```
cnquery run k8s --namespaces "mondoo" -c "asset{ name platform }" 
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
...
asset: {
  platform: "k8s-service"
  name: "mondoo/service1"
}
asset: {
  platform: "k8s-service"
  name: "mondoo/service2"
}
...
```

File: providers/k8s/resources/discovery.go (hunk 1 above)
Comment (imilchev) [reply providers/k8s/resources/discovery.go]: I think you are right, this should be `k8s-service`

**Review by imilchev — APPROVED** [review]

**Pull request merged**